docs: standardize text formatting (#2950)
apollographql/federation: docs/source/quickstart/local-subgraphs.mdx

(As shown, you now pass your `typeDefs` and `resolvers` to `buildSubgraphSchema`.)

> And that's it! You can now perform all of the same subgraph setup on your own service (schema publishing, introspection, etc.) that you did with the Apollo-hosted services we used in the previous parts. Refer to those parts for guidance.

Reviewer: I like the exclamation point for the congratulatory sentence, but I don't like `And that's it`. Maybe just `Congratulations`? 

Or should we avoid congratulatory sentences?

```suggestion
Congratulations! You can now perform all of the same subgraph setup on your own service (schema publishing, introspection, etc.) that you did with the Apollo-hosted services we used in the previous parts. Refer to those parts for guidance.
```

Author: I think some congratulations are alright, particularly at the end of lengthy quickstarts.
